chapters/fundamentals.txt

```diff
@@ -1649,3 +1629,30 @@ include::{generated}/api/structs/VkRect2D.txt[]
 
 include::{generated}/validity/structs/VkRect2D.txt[]
 --
+
+
+// VkStructureType is pretty long,
+// so keep it as a last chapter section for readability.
+=== Structure Types
+
+[open,refpage='VkStructureType',desc='Vulkan structure types (pname:sType)',type='enums']
+--
+Each enumerant of elink:VkStructureType corresponds to a particular
```

Comment:
Please change to "Each enumerant corresponds to a specific structure type, whose pname:sType member contains the enumerant value." People had a hard time understanding this sentence.

With that exception I think people are generally OK with this restructuring, though they want a little longer for review. Thanks.

Reply:
IDK, there's a significant risk that would lead to more wordsmithing. I think I am just gonna revert it to the original.

Though it somewhat bothers me it says "value" instead of "enumerant".